Information/CenterOfMass.FCMacro

```diff
@@ -1065,18 +1205,31 @@ class CenterofmassWidget(QtGui.QWidget):
             selfsol.spinDens.setPalette(pal)
 
     def on_pushButton_toPreferences(self):
-        gui.runCommand('Std_DlgParameter',0)
+        gui.runCommand('Std_DlgParameter', 0)
 
     def on_pushButton_copyToClipboardCdG(self):
         """Copy Vector to clipboard (in FreeCAD Standard unit system)"""
-        string = VALUE_DELIMITER.join(str(self.TotalCoM[axis]) for axis in range(3))
+        string = VALUE_DELIMITER.join(str(self.convert_length(self.TotalCoM[axis])) for axis in range(3))
```

Comment:
Note that the button tooltip says 'Copy to clipboard ({FreeCAD unit system})', so this won't be consistent if you `convert_length`

Reply:
Can we then change the tooltip? it would make more sense to copy the number that is currently displayed, right?